hr_time/translations/de.csv

```diff
@@ -75,4 +75,20 @@ No time tracking,Keine Zeiterfassung
 Employee,Mitarbeiter*in
 Employee Checkin,Zeiterfassung
 "Documents and Events","Dokumente und Protokolle"
-Standard full-time 40 hours,Standard Vollzeit 40 Stunden
\ No newline at end of file
+Standard full-time 40 hours,Standard Vollzeit 40 Stunden
+Timestamp,Zeitstempel
+Time,Zeit
+Task Description,Aufgabenbeschreibung
+Task,Aufgabe
+Action,Aktion
+No employee ID found for the current user,Keine Mitarbeiter-ID für den aktuellen Benutzer gefunden
+Worklog,Arbeitsbericht
+Add Worklog,Arbeitsbericht hinzufügen
+Worklog added successfully,Arbeitsbericht erfolgreich hinzugefügt
+Worklog created successfully,Arbeitsbericht erfolgreich erstellt
+Failed to add Worklog,Konnte kein Arbeitsbericht hinzufügen
+Worklog description cannot be empty,Arbeitsbericht-Beschreibung kann nicht leer sein
+Enter complete detail,Vollständige Details eingeben
+Unknown action provided,Unbekannte Aktion angegeben
+"WARNING: You have no worklogs today. Please create at least one worklog before checking out.","WARNUNG: Sie haben heute noch keinen Arbeitsbericht. Bitte erstellen Sie mindestens einen, bevor Sie auschecken."
```

Comment:
Sounds good. Whereas I am not 100% sure what the "Unknown action provided" error is for or when it is shown - probably "Aktion unbekannt" instead of "Unbekannte Aktion angegeben". Can you quickly tell me what the user needs to do for this error to be shown?

Reply:
Ok. I will use "Aktion unbekannt".

The "Unknown action provided" is triggered if an action is chosen other than:
"Start of work ","Break" or "End of Work".

Although such an action is not present/selectable in the "Action" dropdown (in the UI), it is an extra code of error-proofing the backend in case tools other than the UI are used to set arbitrary action choice. eg. "Finish"